proxy_server.py

```diff
@@ -58,12 +56,14 @@ class AddProxyServerHandler(webapp2.RequestHandler):
 
   """Handler for adding new proxy servers."""
 
-  @admin.RequireAdmin
+  @admin.OAUTH_USER_SCOPE_DECORATOR.oauth_required
+  @admin.RequireAppAndDomainAdmin(admin.OAUTH_USER_SCOPE_DECORATOR)
```

Comment:
I would hope to see something simple like one decorator like below, which would tell me exactly what check is required before processing this method. 
@admin.RequireAppAndDomainAdmin

Instead, the use of the two decorators here is really not intuitive.  I have to go back and see where they are defined to understand what is happening.